tn_api_hotel.yml

```diff
@@ -239,89 +312,449 @@ components:
   schemas:
     HotelSearchResponse:
       properties:
-        available_hotels:
-          type: array
-          items:
-            type: object
-            properties:
-              hotel_id:
-                type: string
-                example: 'ndsf6sjf6sdf7sdf6'
-                description: An identifier to refer to the hotel for searching room availability
-              hotel_name:
-                type: string
-                example: 'Le Blue'
-                description: Name of the hotel
+          hotel_id:
+              type: string
+              example: 'ndsf6sjf6sdf7sdf6'
+              description: An identifier to refer to the hotel for searching room availability
+          hotels:
+              type: array
+              description: A list of hotels available based on the search request details.
+              items:
+                  type: object
+                  properties:
+                      location_iata:
+                          type: string
+                          description: The location of the hotel.
+                          example: 'LON'
+                      location_full:
+                          type: string
+                          description: The full location name as provided by the GDS.
+                          example: "London, England"
+                      rating:
+                          type: int
+                          description: The hotel rating
+                          example: 4
+                      hotel_chain:
+                          type: string
+                          description: The refence for the hotel chain.
+                          example: 'GI'
+                      hotel_code:
+                          type: string
+                          description: The reference for the hotel code
+                          example: '35573'
+                      hotel_name:
+                          type: string
+                          description: The name of the hotel
+                          example: 'Cambridge Suites Hotel Halifax'
+                      minimum_charge:
```

Comment:
I believe this is the minimum daily rate for those days (for example: twin beds, nonrefundable, while the maximum_charge is suite w/ jacuzzi, refundable). We should also return a minimum_stay_price = minimum_charge * number_of_nights.